example/DSSAT/PETPT.java

```diff
@@ -0,0 +1,21 @@
+double ALBEDO;
```

Comment:
This is not Java. You need to create a function. Something like

public static double methodName(...) {
   // body
}